pitop/miniscreen/core/canvas.py

```diff
@@ -1,9 +1,9 @@
 from os.path import isfile
-from PIL import (
-    Image,
-    ImageDraw,
-    ImageFont,
+from .image_helper import (
```

Comment:
Careful with this - we should merge in #172 first!

Reply:
yep, definitely...